API.md

```diff
@@ -85,89 +85,115 @@ fs.writeFileSync('./style.json', JSON.stringify(migrate(style)));
 
-Validate a Mapbox GL Style as a JSON object against the given
-style `reference`. Returns results in the same format as
-`validate`.
+Validate a Mapbox GL style against the style specification. This entrypoint,
+`mapbox-gl-style-spec/lib/validate_style.min`, is designed to produce as
+small a browserify bundle as possible by omitting unnecessary functionality
+and legacy style specifications.
 
 
 ### Parameters
 
-| parameter | type   | description       |
-| --------- | ------ | ----------------- |
-| `style`   | Object | a Mapbox GL Style |
+| parameter     | type   | description                                                                                             |
+| ------------- | ------ | ------------------------------------------------------------------------------------------------------- |
+| `style`       | Object | The style to be validated.                                                                              |
+| `[styleSpec]` | Object | _optional:_ The style specification to validate against.     If omitted, the latest style spec is used. |
 
 
 ### Example
 
 ```js
-var fs = require('fs');
-var validate = require('mapbox-gl-style-spec').validate;
-var spec = require('mapbox-gl-style-spec');
-var style = require('./style.json');
-var errors = validate.parsed(style, spec.v7);
+  var validate = require('mapbox-gl-style-spec/lib/validate_style.min');
+  var errors = validate(style);
 ```
 
 
-**Returns** `Array.<Object>`, an array of errors
+**Returns** `Array.<ValidationError>`, 
+
+
+### `exports(a)`
+
+Turn jsonlint-lines-primitives objects into primitive objects
+
+### Parameters
+
+| parameter | type | description               |
+| --------- | ---- | ------------------------- |
+| `a`       | alu  | potentially-bundled value |
+
+
 
+**Returns** `Object`, an unbundled value
 
-### `validate.latest(style)`
 
-Validate a Mapbox GL Style given a JSON object against the latest
-version of the style spec. Returns results in the same format as
-`validate`.
+### `exports`
+
+Validate a style against the latest specification. This method is optimized
+to keep its bundle size small by refraining from requiring jslint or old
+style spec versions.
+
+
+### `exports(, [options.key], , , , , , )`
+
 
 
 ### Parameters
 
-| parameter | type   | description       |
-| --------- | ------ | ----------------- |
-| `style`   | Object | a Mapbox GL Style |
+| parameter       | type               | description  |
+| --------------- | ------------------ | ------------ |
+| ``              | ption              |              |
+| `[options.key]` | string             | _optional:_  |
+| ``              | ptions\.valu       |              |
+| ``              | options\.valueSpec |              |
+| ``              | options\.style     |              |
+| ``              | options\.styleSpec |              |
+| ``              | options\.layer     |              |
+| ``              | ptions\.objectKe   |              |
```

Comment:
This method :point_up_2: is improperly named. It looks like one of the `validate*` methods (i.e. `validateLayer`, `validateLayoutProperty`)